shadow-dom/reference-target/tentative/label-for.html

```diff
@@ -33,7 +33,7 @@
 
 <!-- 2. Label for attribute targets a custom element using shadowrootreferencetarget inside multiple layers of shadow roots -->
 
-<label for="x-outer2">Input 2</label>
+<!-- <label for="x-outer2">Input 2</label>
```

Comment:
Can you revert the commenting out in this file?

Reply:
Woops woops, reverted.